components/ui/LinkItem.jsx

```diff
@@ -0,0 +1,14 @@
+import Image from 'next/image'
+
+const LinkItem = ({children}) => {
+  return (
+    <div className='flex items-center gap-1 group cursor-pointer'>
```

Comment:
use Link component from Next js instead of a div tag, because this is going to be a link